src/client/components/features/CampagneApprentissage/CampagneApprentissageJeunes/RaisonsDeChoisirApprentissage/RaisonsDeChoisirApprentissage.module.scss

```
@@ -0,0 +1,82 @@
+@use "@styles/utilities";
+
+$padding-horizontal: 2rem;
+
+%container {
+  padding: 1.75rem $padding-horizontal;
+
+  @include utilities.media(large) {
+    padding: 5rem 1rem;
+  }
+}
+
+.raisons {
+  @extend %container;
+  background-color: utilities.$color-background-white-lilac;
+
+  & h2 {
+    @extend %subheading;
+    color: utilities.$color-primary;
+    margin-bottom: 1.5rem;
+
+    @include utilities.media(large) {
+      margin-bottom: 2.5rem;
+      text-align: center;
+    }
+  }
+
+  & ul {
+    $min-width-tuile: 144px;
+    $gap-between-tuiles: 1rem;
+
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap-between-tuiles;
+
+    @include utilities.media(large) {
+      gap: 2.5rem;
+    }
+
+    & li {
+      flex: 1;
+      min-width: $min-width-tuile;
+    }
+
+    & .tuile {
+      height: 100%;
+    }
+
+    > li .tuile {
+      border: 1px solid utilities.$color-primary;
+
+      @include utilities.media(large) {
+        border-width: 2px;
+      }
+    }
+
+    > li:nth-child(even) .tuile {
+      background-color: utilities.$color-on-primary;
+      color: utilities.$color-primary;
+    }
+
+    > li:nth-child(odd) .tuile {
+      background-color: utilities.$color-primary;
+      color: utilities.$color-on-primary;
+    }
+
+    $three-columns: calc(3 * $min-width-tuile + 2 * $padding-horizontal + 2 * $gap-between-tuiles - 1px);
+    $two-columns: calc(2 * $min-width-tuile + 2 * $padding-horizontal + $gap-between-tuiles);
+
+    @media ((max-width: $three-columns) and (min-width: $two-columns)) {
```

nitpick: parce que je suis perturbé par pas grand chose 😁 
 ```suggestion
    @media ((min-width: $two-columns) and (max-width: $three-columns)) {
```